umbraco/Umbraco-CMS
# Content items must prevent saving while image upload is in progress

* Emit JS event when an upload of an image starts & stops
* Listen for events in content edit JS controller to change the state of the buttons to be 'busy' to avoid page/node being saved whilst a large image is uploaded & or using a slow connection

## Test Notes
* Open Chrome DevTools & set the network connection to **Slow 3G**
* Upload an image in the RTE (drag & drop from your computer in)
* Verify that you are shown a notification about saving being disabled
* Verify that the buttons change to the busy state when uploading
* Verify the buttons change to the success tick once the image finished uploading
* Try with one or more images

Fixes [AB#2727](https://umbraco.visualstudio.com/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/2727)
